### DISCUSSION TITLE
Fix MyQ garage door opener issue through Ring account association?

### DISCUSSION THREAD

**Discussion by AdaminOhio** [dgreif/ring#1376] [Ideas]
I am not a developer so I apologize if this is a dumb idea...

I recently noticed in the Ring app on iPhone that you can associate your MyQ account with Ring and then see your garage door and open and close it through the Ring app.  Recently the HomeBridge community that utilized the MyQ plugin lost the ability the functionality to open and close and monitor status of garage door through the MyQ Plugin in HomeBridge due to the Parent Company for MyQ shutting down their external API's (I think I said that sorta right...).

With the recent ability to associate MyQ in the Ring App and add my garage door opener in Ring, is it possible for some Great Mind to update the Ring HomeBridge Plugin to pull in that "association" so it can be identified as an accessory in the Ring HomeBridge app, and essentially get MyQ garage door openers working in HomeBridge again?

I certainly hope this is a possibility.  It has been very sad to lose MyQ garage door functionality through HomeBridge.  Please Ring HomeBridge Plugin Owner(s)!

**Comment by dgreif** [comment]
See #1338
